src/Billing/BillingSettings.cs

```diff
@@ -6,6 +6,7 @@ public class BillingSettings
     public virtual string StripeWebhookKey { get; set; }
     public virtual string StripeWebhookSecret { get; set; }
```

Comment:
This can be part of another PR, but we can go ahead with deleting this oldest secret, `StripeWebhookSecret`.